_pages/sponsors.md

```diff
@@ -1,13 +1,33 @@
 ---
 layout: page
-title: Sponsors
-subtitle: Sponsors
 permalink: /sponsors/
 ---
 
 <div class="pretty-links">
 
-We're currently looking for sponsors!
+# Sponsors
+
+<br>
+
+![](/assets/img/sponsors/MEF.png)
```

Comment:
The logos don't load for me when I test locally and I don't see them in assets/img in the ek_add_sponsors branch.